public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -883,10 +881,12 @@ export class DashboardModel {
     return rowPanels;
   }
 
+  /** @deprecated */
```

Comment:
Maybe we should also add a deprecation warning to these functions too?

Reply:
good idea, added!